orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/AbstractInstancesCheckTask.groovy

```diff
@@ -101,10 +101,7 @@ abstract class AbstractInstancesCheckTask extends AbstractCloudProviderAwareTask
         }
 
         seenServerGroup[name] = true
-        Collection<String> interestingHealthProviderNames = stage.execution.appConfig.interestingHealthProviderNames as Collection
-        if (interestingHealthProviderNames == null) {
```

Comment:
We ran some tests with additional instrumentation and agree that `stage.context.interestingHealthProviderNames` is the right place to look, and the `stage.context.appConfig` references are extraneous.